### DISCUSSION TITLE
Experimental $ref syntax error in nuxt 3 (reactivityTransform)

### DISCUSSION THREAD

**Discussion by dajpes** [vuejs/language-tools#1065] [Q&A] [answered]
Hey you all!

So I've been playing around nuxt 3 with the reactivityTransformm, but I get this error (the code is working ok):

nuxt.config.js
![image](https://user-images.githubusercontent.com/11248816/158864503-f2b2f832-a55e-4f36-8902-bbcfe1cee100.png)

App.vue

![image](https://user-images.githubusercontent.com/11248816/158864595-abcdeaf6-ab02-4421-82a5-5f69e2eead8d.png)

**Comment by johnsoncodehk** [comment, marked as answer]
Hi @dajpes, see #521.
